### ISSUE TITLE
Consider storing logged in status after page refresh

### ISSUE THREAD

**Issue by IgorShadurin** [fairDataSociety/fairdrive-theapp#501]
Fairdrive's current behavior resets the logged in status whenever a user updates the page. It would be beneficial to reconsider this and possibly introduce a feature to retain the logged in status for a certain period. Here are a few things to determine:

- Evaluating if storing the logged in status is secure for Fairdrive.
- Ensuring the compatibility of all existing login methods such as username-password, Blossom, and Metamask.
- Determining the ideal time period to store the login status.
- Figuring out the best method to store this information (e.g., in localStorage/sessionStorage? Should it be encrypted or stored in raw format?)
- Identifying any libraries that could assist in implementing this feature.

This improvement could significantly enhance the user experience while utilizing the app.

@tomicvladan @tfius

**Comment by tomicvladan** [comment]
I don't think we can achieve this without sacrificing security. In order to keep user logged in after a page refresh/opening new page/ restarting browser etc., password must be stored somewhere locally. But it can't be encrypted, because there is no way to decrypt it. So if there were be any vulnerabilities in the app, then it would be possible to steal the password.

**Comment by tamas6** [comment]
related: #491
